### PR TITLE
Mapintegrated context cards

### PR DESCRIPTION
**Pull request by Tehsurfer**
- Disable the context card
 - Add an emit for facet labels

### CODE REVIEW

File: src/components/DatasetCard.vue [hunk 1]
```diff
@@ -168,6 +168,7 @@ export default {
     propogateCardAction: function(action){
       EventBus.$emit("PopoverActionClick", action)
       this.$emit('contextUpdate', action)
```
Comment (alan-wu) [inline src/components/DatasetCard.vue:170]: This should either be removed or replaced by` EventBus.$emit("contextUpdate", action) ` completely.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good in general. I want to combine the emit and eventbus of contextUpdate into one call.
We should be able to combine the following as well
   EventBus.$on('anatomyFound')
   EventBus.$on('labelsFound')
 But we will leave that for later.

File: src/components/DatasetCard.vue [hunk 2]
```diff
@@ -167,7 +167,6 @@ export default {
     },
     propogateCardAction: function(action){
       EventBus.$emit("PopoverActionClick", action)
-      this.$emit('contextUpdate', action)
```
Comment (alan-wu) [inline src/components/DatasetCard.vue:170]: Thanks for removing that but SidebarContent hasgot an event handler on line 37.

Comment (Tehsurfer) [reply src/components/DatasetCard.vue:170]: tyty I went ahead and removed all the context card variables and references

**Pull request merged**